hack/dump-state.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+
+function RunCmd {
+    cmd="$@"
+    echo "Command: $cmd"
+    $cmd
+    if [ "x$?" != "x0" ]; then
+        echo "Command failed: $cmd Status: $?"
+    fi
+}
+
+cat <<EOF
+=================================
+     Start of HCO state dump         
+=================================
+
+========
+HCO Pods
+========
+
+EOF
+
+RunCmd "$CMD get pods -n kubevirt-hyperconverged -o json"
+
+cat <<EOF
+===============
+HCO Deployments
+===============
+EOF
+
+RunCmd "$CMD get deployments -n kubevirt-hyperconverged -o json"
+
+cat <<EOF
+========================
+HCO operator related CRD
+========================
+EOF
+
+RELATED_OBJECTS=`${CMD} get hyperconvergeds.hco.kubevirt.io hyperconverged-cluster -n kubevirt-hyperconverged -o go-template='{{range .status.relatedObjects }}{{if .namespace }}{{ printf "%s %s %s\n" .kind .name .namespace }}{{ else }}{{ printf "%s %s .\n" .kind .name }}{{ end }}{{ end }}'`
```

Comment:
You may be able to use the command I mentioned above to get the component operators statuses too (haven't tried yet).

Reply:
ok